### ISSUE TITLE
Staging Review finding: Missing focus management when providing details about conditions

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#108130]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Disability Benefits Experience Team 2
**Product:** Disability Comp Application (21-526 EZ)
**Feature:** Updated 0781 Flow

## Findings details
**VA.gov Experience Standard - issue:** [User experiences predictable interactions with components and patterns.](https://design.va.gov/about/experience-standards/#user-can-identify-an-element-and-its-state)
**VA.gov Experience Standard - category:** [Usability](https://design.va.gov/about/experience-standards/#category-usability)
**Launch-blocking:** Yes
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

This is outside the scope of the review but was encountered during the review, and would be considered a launch-blocker. Not sure if this goes to your team or someone else, but please make sure it gets prioritized!

When you've entered the conditions you're claiming for your submission and you iterate through them to provide details about each, there doesn't appear to be any change of keyboard focus on page transitions --- eg. when providing details on "Hearing loss" and then pressing Continue to go on to "Throat cancer," focus stays on the Continue button rather than moving back to the top of the page.

There are a couple of consequences to this:
- For sighted users whose viewport isn't large enough or isn't positioned to see the heading at the top, it would be easy to miss that any page transition happened at all. A user may think they need to wait for something to happen, or may think they need to press Continue again and inadvertently skip a page. (This is illustrated in the included video.)
- For screen reader users, nothing is announced when pressing Continue. Silence can be confusing and disorienting, and users will have to explore the page to determine what's happened (or may guess wrong and try to press Continue again).

<details><summary>Link, screenshot or steps to recreate</summary>

[video 80848-Brian-Step2FocusManagement]

</details>

## Recommended action

Double-check focus management behavior throughout the form flow. It's a BIG form and something may have gotten changed accidentally in the course of other work.

## References

- **Accessibility Defect Severity:** [1: Critical. Must be fixed before launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-1)
- **WCAG Success Criteria:** N/A
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by etanb** [comment]
Going to un-assign myself from this ticket until we can formally incorporate it into our new in-progress Accessibility workflow: https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/vsa/teams/disability-experience/team-docs/a11y-process.md 

cc @comaurice
